pkg/cmd/step/buildpack/test_data/step_buildpack_apply/go_inherit/Jenkinsfile

```diff
@@ -23,7 +23,7 @@ pipeline {
             checkout scm
             sh "make linux"
             sh "export VERSION=$PREVIEW_VERSION && skaffold build -f skaffold.yaml"
-            sh "jx step post build --image $DOCKER_REGISTRY/$ORG/$APP_NAME:$PREVIEW_VERSION"
+            sh "jx step post build --image $DOCKER_REGISTRY/$DOCKER_REGISTRY_ORG/$APP_NAME:$PREVIEW_VERSION"
```

Comment:
We probably don't need to be changing the `cmd/step/buildpack` tests - these are really testing the generation of Jenkinsfiles from build packs for static master setups, and in static masters, `ORG` is set by Jenkins, and `DOCKER_REGISTRY_ORG` is not being set, and we're not going to waste the effort in trying to change that since we're removing support for static masters in the near future. That said, we're not going to be changing the build packs any time soon because build packs aren't versioned, meaning existing static master users are still going to be picking up `master` in the build packs repos, so..yeah. Messy.

Regardless - it doesn't _really_ matter here since these tests are using buildpacks specified here in the test, not the ones from the build packs repos, so it's ok to change these since you already did. =)